src/MessageHandler/ProcessFileHandler.php

```diff
@@ -122,13 +122,14 @@ public function __invoke(ProcessFile $processFile)
             );
             $file->setPhysicalFilePath($newFileDestination);
             unlink($filePath);
-            rmdir(dirname($filePath));
         } catch (\Exception $exception) {
             $this->logger->error(sprintf('Unable to add file to datastore. Message: "%s"', $exception->getMessage()), $loggingContext);
             $file->setStatus(File::FILE_ERROR);
             $this->entityManager->flush();
             return;
         }
+        
+        @rmdir(dirname($filePath));
```

Comment:
This had to be done, because if a folder in not empty, this would thrown an error. And fail the handler.